cosmo_tester/test_suites/test_security/security_test_base.py

```diff
@@ -118,8 +134,39 @@ def _update_manager_blueprint(self):
             for key, value in props.items():
                 patch.set_value(key, value)
 
+    def get_userstore_users(self):
+        return None
+
+    def get_userstore_groups(self):
+        return None
+
+    def get_userstore_settings(self):
+        settings = {}
+        users = self.get_userstore_users()
+        if users:
+            settings['users'] = users
+        groups = self.get_userstore_groups()
+        if groups:
+            settings['groups'] = groups
+
+        return settings
+
+    def _update_userstore_file(self):
+        userstore_settings = self.get_userstore_settings()
+        with util.YamlPatcher(self.test_userstore_file_path) as patch:
+            for key, value in userstore_settings.items():
+                patch.set_value(key, value)
+
     def get_manager_blueprint_additional_props_override(self):
-        return {}
+        overrides = {}
+        rest_plugins = self.get_rest_plugins()
+        if rest_plugins:
+            overrides = {'{0}.plugins'.format(REST_PLUGIN_PATH): rest_plugins}
```

Comment:
As mentioned in the previous comment, `REST_PLUGIN_PATH` should contain `.plugins` in it (meaning its value should remain as `PLUGINS_PROP_PATH` was, maybe just renamed if your prefer).
Also, if you only use `MANAGER_HOST_PARAMETERS` for userdata, change it to include `.user_data` and rename it to `USER_DATA_PATH`. That way you can avoid the explicit prefixing added in this method.